### DISCUSSION TITLE
Awesome Cheatsheets by LeCoupa

### DISCUSSION THREAD

**Discussion by anilpenumacha** [community/community#49052] [GitHub Education]
Hi, I am not aware how to use this GitHub Repo, Can some one assist me on the same.

GitHub Repo : https://github.com/LeCoupa/awesome-cheatsheets

Thanks in advance.

**Comment by Anmol-Baranwal** [comment]
What do you want to do? Clearly state. 

> If you want to contribute in that repo, then you can follow [this](https://docs.github.com/en/get-started/quickstart/github-flow)

**Reply by Anmol-Baranwal** [reply]
@anilpenumacha 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
